operations/helm/charts/mimir-distributed/CHANGELOG.md

```diff
@@ -13,6 +13,43 @@ Entries should include a reference to the Pull Request that introduced the chang
 
 ## main / unreleased
 
+* [CHANGE] **breaking change** **Data loss without action.** Enable [zone aware replication](https://grafana.com/docs/mimir/latest/operators-guide/configure/configuring-zone-aware-replication/) for ingesters and store-gateways by default. #2778
+  - If you are **installing** the chart:
```

Comment:
This needs to be in the core documentation instead of the `CHANGELOG.md` file.

Reply:
which core documentation?